ipatests/test_xmlrpc/tracker/base.py

```diff
@@ -76,6 +76,7 @@ def __init__(self, default_version=None):
         self.api = api
         self.default_version = default_version or API_VERSION
         self._dn = None
+        self.attrs = None
```

Comment:
Can you point me to an instance of some method or test accessing the attributes? I think an empty dictionary would be more appropriate here.